packages/lesswrong/server/sql/SqlFragment.ts

```diff
@@ -54,31 +54,138 @@ export const getResolverCollection = (
   return getCollectionByTypeName(type);
 }
 
+function extractFragmentDefinitions(fragmentSrc: string): Record<string, string> {
```

Comment:
Is there a reason this is a shotgun parser rather than using the graphql library's parser?